src/core/actions/categories.js

```diff
@@ -1,13 +1,15 @@
 import {
   CATEGORIES_FETCH,
   CATEGORIES_LOAD,
-  CATEGORIES_FAIL,
 } from 'core/constants';
 
-export function categoriesFetch() {
+export function categoriesFetch({ errorHandlerId } = {}) {
+  if (!errorHandlerId) {
```

Comment:
I wish we could do some kind of meta-programming for JS variable names and write a convenience function for things like this.